sanity.config.ts

```diff
@@ -2,6 +2,21 @@ import { defineConfig } from "sanity";
 import { structureTool } from "sanity/structure";
 import { schemaTypes } from "./schemas";
 import { dataset, projectId } from "./utils/sanity";
+import { StructureBuilder } from "sanity/structure";
+
+const structure = (S: StructureBuilder) =>
+  S.list()
+    .title("Ny i Abakus")
+    .items([
+      S.listItem()
+        .title("Site Settings")
+        .child(
+          S.document().schemaType("siteSettings").documentId("siteSettings")
+        ),
+      ...S.documentTypeListItems().filter(
+        (listItem) => !["siteSettings"].includes(listItem.getId() ?? "")
+      ),
+    ]);
```

Comment:
haven't seen this before so I'm trusting you on this one

Reply:
See my most recent comment, I just copied this part from a guide and it essentially sets one item with just one child, the settings document, and then lists the rest of the items. This way it is not possible to create more "settings" objects.

Reply:
Might look into adding it here actually as it seems vercel will not build this PR since I do a lot of "illegal" stuff even though my types dont complain and it works locally 😓